tests/structuredheadings/clearnumberingstyle.js

```diff
@@ -43,6 +54,15 @@
             bot.getData(),
             "cleared styling from headings"
         );
+
+        bot.execCommand("undo");
+        // this is weird, but the selection API returns the extra (internal) paragraphs
+        // that are created to normalize the selection here
+        assert.areSame(
+          initialHtmlWithSelection.substring(1, initialHtmlWithSelection.length - 1),
```

Comment:
Can we assign this value to a variable before using it? It's not clear to me why we need to do this substring, even with the comment above.